src/test/java/com/emc/object/s3/V2STSBasicTest.java

```diff
@@ -0,0 +1,213 @@
+package com.emc.object.s3;
+
+import com.emc.object.Method;
+import com.emc.object.ObjectConfig;
+import com.emc.object.Protocol;
+import com.emc.object.s3.jersey.S3JerseyClient;
+import com.emc.object.s3.request.PresignedUrlRequest;
+import com.emc.object.util.TestProperties;
+import com.emc.rest.smart.ecs.Vdc;
+import com.emc.util.TestConfig;
+import com.sun.jersey.api.client.Client;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Date;
+import java.util.Properties;
+
+public class V2STSBasicTest extends S3JerseyClientTest{
+    private static final Logger l4j = Logger.getLogger(V2STSBasicTest.class);
+    private static final String SESSION_TOKEN = "Cghuc190ZXN0MRIIaWFtX3VzZXIaFEFST0EzQjFGMDc0OUJFQkIzRDlFIiB1cm46ZWNzOmlhbTo6bnNfdGVzdDE6cm9sZS9yb2xlMSoUQVNJQUI1MTEzMzYwN0FBNzg1QjUyUE1hc3RlcktleVJlY29yZC0zZGE0ZTJlNmMyMGNiMzg2NDVlZTJlYjlkNWUxYzUxODJiYTBhYjQ3NWIxMDg4YWE5NDBmMzIyZTAyNWEzY2Q1OKXTrK2VL1IMZWNzLXN0cy10ZW1waL_l44QG";
+
+    protected S3Config s3ConfigFromProperties() throws Exception {
+        Properties props = TestConfig.getProperties();
```

Comment:
You can simplify this method by calling the super implementation first, then modifying the config.

Reply:
Done

Reply:
extract the duplicated part in the abstract class into a new function